miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -226,6 +226,7 @@
   <miso:list-section id="list_order" name="Requested Orders" target="order" alwaysShow="true" items="${orders}" config="{ poolId: ${pool.id}, platformType: '${pool.platformType.name()}' }"/>
   <miso:list-section-ajax id="list_completion" name="Order Status" target="completion" config="{ poolId: ${pool.id} }"/>
   <miso:list-section id="list_run" name="Runs" target="run" items="${runs}" config="{ poolId: ${pool.id} }"/>
+  <miso:list-section id="list_partition" name="${pool.platformType.partitionName}" target="partition" items="${partitions}" config="{'showContainer': true, 'showPool': false}"/>
```

Comment:
The `Assign Pool` action doesn't make sense on this page, so should probably be omitted when `showPool == false` (in `list_partition.js`)